usagov/vote-gov-drupal
# VOTE-1380 Add touchpoints survey to the footer menu block

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1308

## Description (optional)
Added touch points script to the library yaml file
Add button to the footer for testing

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. lando retune
2. You may have to rebuild styles
cd web/themes/custom/vote_gov
npm run build

### Testing steps
1. Click on the button "How can we improve Vote.gov"
2. Verify that the survey loads
3. Submitting feedback won't work on local. This may need to be tested in the dev environment.
